foreman-proxy/logrotate.conf

```diff
@@ -1,4 +1,7 @@
-/var/log/foreman-proxy/*log {
+/var/log/foreman-proxy/cron.log
```

Comment:
It should be removed then unless you can cite the source, I don't think it does anything.